Service/Request/RequestFactory.php

```diff
@@ -149,6 +147,26 @@ private function getRequestUri(ServiceRequestInterface $serviceRequest): UriInte
         return $this->uriFactory->createUri($baseUrl.$path);
     }
 
+    /**
+     * @param ServiceRequestInterface $serviceRequest
+     * @param EndpointInterface $endpoint
+     * @return mixed
+     */
+    private function getRequestBody(ServiceRequestInterface $serviceRequest, EndpointInterface $endpoint)
+    {
+        $isMethodWithoutBody = in_array($endpoint->getMethod(), self::METHODS_WITHOUT_BODY, true);
```

Comment:
It looks like the temporary variable is required only to make the code "self-describing".
At this point, could it be a separate method `isMethodHasBody`? In this case, no temporary variable is not required and all methods will have a "single responsibility"
And also, should this method belong to the `Endpoint` entity, which should encapsulate all information, required for API calls?

Reply:
I agree about extracting ```isMethodHasBody``` method. I also agree with statement about ```Endpoint```, but in that case we will need to increase major version of https://github.com/auto1-oss/service-api-components-bundle. I can do that if you agree with that.

Reply:
Let’s merge it without moving the method to `Endpoint`